java/src/main/java/com/google/cloud/dataproc/templates/gcs/GCStoGCS.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.dataproc.templates.gcs;
+
+import static com.google.cloud.dataproc.templates.util.TemplateConstants.*;
+
+import com.google.cloud.dataproc.templates.BaseTemplate;
+import java.util.Objects;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.spark.sql.DataFrameWriter;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GCStoGCS implements BaseTemplate {
+
+  public static final Logger LOGGER = LoggerFactory.getLogger(GCStoGCS.class);
+
+  private String projectID;
+  private String inputFileLocation;
+  private String inputFileFormat;
+  private String gcsOutputLocation;
+  private String gcsWriteMode;
+  private String gcsPartitionColumn;
+  private String gcsOutputFormat;
+
+  private String tempTable;
+
+  private String tempQuery;
+
+  public GCStoGCS() {
+
+    projectID = getProperties().getProperty(PROJECT_ID_PROP);
+    inputFileLocation = getProperties().getProperty(GCS_GCS_INPUT_LOCATION);
+    inputFileFormat = getProperties().getProperty(GCS_GCS_INPUT_FORMAT);
+    gcsOutputLocation = getProperties().getProperty(GCS_GCS_OUTPUT_LOCATION);
+    gcsPartitionColumn = getProperties().getProperty(GCS_GCS_OUTPUT_PARTITION_COLUMN);
+    gcsOutputFormat = getProperties().getProperty(GCS_GCS_OUTPUT_FORMAT);
+    gcsWriteMode = getProperties().getProperty(GCS_GCS_WRITE_MODE);
+    tempTable = getProperties().getProperty(GCS_GCS_TEMP_TABLE);
+    tempQuery = getProperties().getProperty(GCS_GCS_TEMP_QUERY);
+  }
+
+  @Override
+  public void runTemplate() {
+    validateInput();
+
+    SparkSession spark = null;
+    LOGGER.info(
```

Comment:
duplicate code
same logger is already present in validateInput()
Please remove it from here.

Reply:
Done